ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml

```
@@ -32,6 +32,17 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+tests:
+- as: lint-aws-plugin-template
```

```suggestion
- as: lint
```

unless you expect to have multiple lint jobs